packages/terra-time-input/tests/wdio/time-input-twelve-hour-mobile-spec.js

```diff
@@ -93,7 +86,7 @@ Terra.describeViewports('Time Input Twelve Hour Mobile', ['medium'], () => {
       browser.keys('123456');
     });
 
-    Terra.it.isAccessible({ rules: ignoredA11y });
+    Terra.it.isAccessible();
```

Comment:
This could be `Terra.it.validatesElement();`

Reply:
Resolved in 009e0a3